frontend/src/components/survey-handler/survey-handler.tsx

```diff
@@ -173,7 +176,11 @@ export const SurveyHandler = (props: SurveyPageProps) => {
             </Heading>
 
             <Box mt={3} color={colorPallettes.secondary}>
-              <span>SELECT UP TO 1 OPTION</span>
+              <span>
+                {question.question_type.name === 'Multiple Choice'
+                  ? 'SELECT ONE OR MORE OPTIONS'
+                  : 'SELECT UP TO 1 OPTION'}
+              </span>
```

Comment:
What do you think about creating a function that returns this? In the future we will have other types of questions and this sentence may change. I suggest an object literal for this, what do you think?